maxim/tests/test_crewai.py

```
@@ -0,0 +1,98 @@
+            
+            # Basic assertions
+            self.assertIsNotNone(result)
+            self.assertIsInstance(result, str)
+            self.assertGreater(len(result), 0)
+
+            print("Crew execution result:", result)
+
+        except Exception as e:
+            self.skipTest(f"Crew execution error: {e}")
+
+    def tearDown(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `tearDown`

Add return type annotation: `None`

(ANN201)